projects/frontend/data-pipelines/gui/config/nginx.conf

```diff
@@ -0,0 +1,11 @@
+server {
+    listen 8091;
+    root /usr/share/nginx/html;
+    large_client_header_buffers 4 32k;
+    add_header Strict-Transport-Security "max-age=0; includeSubDomains" always;
+
+    location / {
+        try_files $uri $uri/ /index.html =404;
+        add_header Strict-Transport-Security "max-age=0; includeSubDomains" always;
```

Comment:
That's a duplicate. It should not be necessary since it's already defined above at server level.